src/03-setup/macos.md

```diff
@@ -5,7 +5,6 @@ All the tools can be install using [Homebrew]:
 [Homebrew]: http://brew.sh/
 
 ``` console
-$ brew cask install gcc-arm-embedded
```

Comment:
Do you still need this for gdb?

Reply:
Yes, you are right; this is still needed for GDB. I'll add it back.